dirsrvtests/tests/suites/ldapi/ldapi_test.py

```diff
@@ -0,0 +1,160 @@
+import logging
+import pytest
+import os
+import subprocess
+from lib389._constants import DEFAULT_SUFFIX, DN_DM
+from lib389.idm.user import UserAccounts
+from lib389.ldapi import LDAPIMapping
+from lib389.ldap_objs import DSContainer
+from lib389.topologies import topology_st as topo
+from lib389.tasks import LDAPIMappingReloadTask
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
Test needs a marker to state that it adds/removes users, especially because your useradd command uses 1001 and 1002 as uids which as low numbers *may* already exist on the system.

Reply:
I changed it to  500[1-3]

Reply:
I still think it needs a "ack" marker, similar to what @droideck did recently for the dd/log test since this will change/affect your system. It also probably needs a "are we root" check too.

Reply:
Bump on this one.

Reply:
I'll remove the log...